### PR TITLE
modified example and readme to use a more maximalist starship prompt config

### PR DESCRIPTION
**Pull request by nataziel**
Only small changes, feel free to reject or suggest changes, just thought I'd send in something that I've been using. I've also submitted this config to the main starship repo presets.

My preview image is much less nice than the previous one that showed all the different flavours but I don't really know how to recreate that image nicely haha.

### CODE REVIEW

File: assets/preview.png
Comment (isabelroses) [inline assets/preview.png:1]: Please use webps and use [catwalk](https://github.com/catppuccin/toolbox/tree/main/catwalk) to generate the previews.

Comment (nataziel) [reply assets/preview.png:1]: Thanks for the feedback, have changed that now :)

**Comment by JoshPaulie** [comment]
Thanks for the PR! 🤩

Your additions to the config really give the user a sense of what is possible with Starship. It's a great reference that I'd like to incorporate in some way. I can tell you worked hard on it, and the final result looks great. I'll be using it myself.

That being said, the additions are outside of the scope for this repo. This is meant to provide an easy copy-and-paste reference for incorporating Catppuccin into Starship. Previous iterations of a Starship port by others were denied due to them being too opinionated. This would most certainly be considered an opinionated config.

The current config only serves as an example of how to use user-defined colors. **If your submission to the Starship themes list is accepted, I think a link to it would be an awesome addition to the README.md** 😄

**Comment by zjr** [comment]
Got to this PR hoping to find exactly this kind of thing, great timing for me! Sorry if this is a little comment-spammy, just wanted to say thanks to @nataziel! Hope it gets merged to Starship. I'll try testing MacOS later, might help you along with their checklist?

**Comment by nataziel** [comment]
@JoshPaulie completely understood. Hopefully it gets accepted by starship :)

**Comment by JoshPaulie** [comment]
@nataziel hello!
We've started a show-and-tell section in the Discussions portion of this repo. Your preset would be a great addition!

**Pull request closed without merging**